.gitlab-ci.yml

```diff
@@ -13,7 +13,7 @@ stages:
 
 variables:
   GIT_STRATEGY:                    fetch
-  GIT_DEPTH:                       100
+#  GIT_DEPTH:                       100
```

Comment:
Best would be if we could make this `1`.

Reply:
I'm just testing whether having it set it is causing the issue we are seeing